aspnet-core/xml/ns-Microsoft.AspNetCore.Server.HttpSys.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Server.HttpSys">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains types that are used to run an ASP.NET Core app with HTTP.sys. HTTP.sys will be used as the <see cref="T:Microsoft.AspNetCore.Hosting.Server.IServer" />. HTTP.sys is only supported on Windows Operating Systems.</summary>
```

Comment:
We try to avoid future tense, but in this case it might be justified. Feel free to ignore.

```suggestion
    <summary>Contains types that are used to run an ASP.NET Core app with HTTP.sys. HTTP.sys is used as the <see cref="T:Microsoft.AspNetCore.Hosting.Server.IServer" />. HTTP.sys is only supported on Windows Operating Systems.</summary>
```